.github/actions/action-codeql/action.yml

```
@@ -0,0 +1,54 @@
+        languages: ${{ inputs.codeql-language }}
+        build-mode: ${{ inputs.codeql-buildmode }}
+        queries: ${{ inputs.codeql-query }}
+    - if: inputs.codeql-buildmode == 'autobuild'
+      name: Build using Autobuild
+      uses: github/codeql-action/autobuild@v3
+      with:
+        working-directory: ${{ inputs.path }}
+    - name: Perform CodeQL analysis for ${{ inputs.codeql-language }}
+      uses: github/codeql-action/analyze@v3
+      with:
+        category: "/language:${{ inputs.codeql-language }}-/path:${{ inputs.path }}"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)